yamlReader.py

```diff
@@ -0,0 +1,168 @@
+import yaml
+import yamlordereddictloader
+import start
+
+
+yamlName = start.yamlName()
+
+
+def getDefault():
+    defaultDay = -1
+    with open(yamlName, "r") as f:
+        # yaml file opening and loading
+        y = yaml.load(f, Loader=yamlordereddictloader.Loader)
+        for (attribute, value) in y.items():
+            # fetching employees list
+            if (attribute == "default_date_day"):
+                defaultDay = int(value)
+    return defaultDay
+
+
+def getEmployeesList():
+    with open(yamlName, "r") as f:
+        # yaml file opening and loading
+        y = yaml.load(f, Loader=yamlordereddictloader.Loader)
+        for (attribute, value) in y.items():
+            # fetching employees list
+            if (attribute == "employees"):
+                employeesList = []
+                for employee in value:
+                    for (attribute, value) in employee.items():
+                        if (attribute == "employee"):
+                            employeesList.append(value)
+    return employeesList
+
+
+def getRulesList():
+    with open(yamlName, "r") as f:
+        # yaml file opening and loading
+        y = yaml.load(f, Loader=yamlordereddictloader.Loader)
+        for (attribute, value) in y.items():
+            # fetching rules list
+            if (attribute == "wage_rules"):
+                rulesList = []
+                for rule in value:
+                    for (attribute, value) in rule.items():
+                        if (attribute == "rule"):
+                            rulesList.append(value)
+    return rulesList
+
+
+def getYaml():
+    return yamlName
+
+
+def getHollidays():
+    # getting string representations in string form
+    hollidayList = []
+    with open(yamlName, "r") as f:
+        y = yaml.load(f, Loader=yamlordereddictloader.Loader)
+        for (attribute, value) in y.items():
+            if attribute == "bank-hollidays":
+                for day in value:
+                    hollidayList.append(day)
+                break
+    return hollidayList
+
+
+def getEmployees():
+    with open(yamlName, "r") as f:
+        # yaml file opening and loading
+        y = yaml.load(f, Loader=yamlordereddictloader.Loader)
+        for (attribute, value) in y.items():
+            # fetching employees list
+            if (attribute == "employees"):
+                employeesList = []
+                for employee in value:
+                    employeesList.append(employee)
+    return employeesList
+
+
+def getHourlyWage():
+    with open(yamlName, "r") as f:
+        # yaml file opening and loading
+        y = yaml.load(f, Loader=yamlordereddictloader.Loader)
+        for (attribute, value) in y.items():
+            # fetching hourly wage
+            if (attribute == "hourly_wage"):
+                hourlyWage = value
+    return hourlyWage
+
+
+def getHourlyWageAfter5pm():
```

Comment:
Are these in use any more?

If not, delete them.

Generally speaking, delete everything you don't use.

Reply:
Obsolete functions deleted